src/messages/manifestjson.js

```diff
@@ -795,3 +795,13 @@ export const VERSION_FORMAT_INVALID = {
     https://mzl.la/3h3mCRu (MDN Docs) for more information.`),
   file: MANIFEST_JSON,
 };
+
+export const INSTALL_ORIGINS_REQUIRED = {
+  code: 'INSTALL_ORIGINS_REQUIRED',
+  message: i18n._(
+    'The "install_origins" property is required in the manifest.'
+  ),
+  description: i18n._(oneLine`The "install_origins" property is required in the
+    manifest for self-hosted versions in Manifest Version 3 and above.`),
```

Comment:
Should we wait until we have MDN docs live to include the relevant URL, so that we don't have to change the string and redo translations ?

Reply:
Sure